edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/KubernetesDeploymentMapperTest.cs

```diff
@@ -166,7 +166,8 @@ public void PvcMappingForVolumeNameVolume()
             Assert.True(pod != null);
             var podVolume = pod.Spec.Volumes.Single(v => v.Name == "elephant");
             Assert.NotNull(podVolume.PersistentVolumeClaim);
-            Assert.Equal(module.Name + "a-volume", podVolume.PersistentVolumeClaim.ClaimName);
+
+            Assert.Equal(module.Name + "-a-volume", podVolume.PersistentVolumeClaim.ClaimName);
```

Comment:
Usually, it is recommended to put some precomputed value to increase the readability of the tests. And if it fails on this line it will be obvious to see what's wrong.
I'll leave it only here but can be applied to all changes below
```suggestion 
            Assert.Equal("module1-a-volume", podVolume.PersistentVolumeClaim.ClaimName);
```

Reply:
En, good point. It seems that the module.Name is a common pattern in the existing test cases. Let's have a work item to improve readability.